### DISCUSSION TITLE
Username Collecting

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#5123] [Questions] [answered]
Hello, I have some pretty well known accounts which I will not be listing I have been username collecting for a while like getting rare usernames not selling them or any of that just like collecting rare usernames for my alts is this okay or bannable?

**Comment by Miodec** [comment, marked as answer]
No, its not allowed

**Reply by ghost** [reply]
where does it say its not allowed people have alternate accounts I use those alternate accounts to stay anonymous
